inference-engine/src/mkldnn_plugin/mkldnn_graph.h

```diff
@@ -92,6 +92,27 @@ class MKLDNNGraph {
     void DropNode(const MKLDNNNodePtr& node);
     void DropDWConvNode(const MKLDNNNodePtr& node);
 
+    /**
+     * @brief Insert Reorder node at the edge-specified location.
+     * The Reorder node must be inserted in case when there are inplace conflicts or the input and output tensor descriptors do not match.
+     * The Reorder node rearranges the elements in memory according to inDesc and outDesc, or does nothing if isOptimized is true.
```

Comment:
`does nothing if isOptimized is true` - it is not fully correct I would say. In fact it reinterprets memory descriptor w/o rearrangement of elements.

Reply:
Thanks! I fully agree, fixed